test/cypress/integration/entry-editor-extension.spec.js

```diff
@@ -12,6 +12,7 @@ import {
   openErrorNotificationTest
 } from './reusable/open-notifications-test'
 import { verifyLocation } from '../utils/verify-location'
+import { openSdkParametersTest } from './reusable/open-sdk-parameters-test'
```

Comment:
I would rather split it to `verifySdkInstallationParameters` and `verifySdkInstanceParameters` to make it explicit what parameters are present.